db/fixtures/miq_policy_sets.yml

```diff
@@ -105,7 +105,24 @@
           description: Initiate SmartState Analysis for Container Image
           action_type: default
           options: {}
-      Condition: []
+      Condition:
+        - name: do not scan image-inspectors image
+          description: Dont scan image-inspectors image
+          modifier: deny
+          expression: !ruby/object:MiqExpression
+            exp:
+              ENDS WITH:
+                field: ContainerImage-name
+                value: "image-inspector"
+            context_type:
+          towhat: ContainerImage
+          file_mtime:
+          guid: e744245a-3d08-11e6-8d39-02422087d789
```

Comment:
Not obvious how, as we don't bring any metadata for images.
All we have is parsing the "registry.com/repo/image:tag" string.
Registry/repo will be configurable.  Can't whitelist hashes as we want to be able to push new image simply by pointing tag to new hash.

Any ideas?

We could look at pod info when we first see a new image string and create a ContainerImage.

- Add column to ContainerImage?  Needs migration.
- Leave ContainerImage alone, attach info to evm event [1]?  Not even sure we can condition on event fields.
- If same image-inspector image is run by both us and users, end result would depend on which pod we saw first.  
  Can make flag mean "have we ever seen this image except in pods we run" => in any order,   meeting user's pod will flip it to false — and then trigger scan.

[1] currently done in https://github.com/ManageIQ/manageiq/blob/95a3906cb6a8990b3766b9062ecd3c00f48eda45/app/models/container_image.rb#L76